lib/simple_url_preview.dart

```diff
@@ -228,18 +246,33 @@ class _SimpleUrlPreviewState extends State<SimpleUrlPreview> {
                 crossAxisAlignment: CrossAxisAlignment.start,
                 children: <Widget>[
                   PreviewTitle(
-                    _urlPreviewData['og:title'],
-                    _textColor,
-                    _titleLines,
-                  ),
+                      _urlPreviewData['og:title'],
+                      _titleStyle == null
+                          ? TextStyle(
+                              fontWeight: FontWeight.bold,
+                              fontSize: 16,
+                              color: _textColor,
```

Comment:
As we will be removing _textColor property, I think it will be good idea to set default color as accentColor.
So default textStyle will become:
TextStyle(
  fontWeight: FontWeight.bold,
  fontSize: 16,
  color: Theme.of(context).accentColor,
)

Similar about descriptionStyle and sitenameStyle.